app/lib/puter.ts

```
@@ -0,0 +1,456 @@
+                read: (path: string) => Promise<Blob>;
+                upload: (file: File[] | Blob[]) => Promise<FSItem>;
+                delete: (path: string) => Promise<void>;
+                readdir: (path: string) => Promise<FSItem[] | undefined>;
+            };
+            ai: {
+                chat: (
+                    prompt: string | ChatMessage[],
+                    imageURL?: string | PuterChatOptions,
+                    testMode?: boolean,
+                    options?: PuterChatOptions
+                ) => Promise<Object>;
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)